index.html

```diff
@@ -0,0 +1,967 @@
+                                        alt=""
+                                    />
+                                    <h4>XBOX SERIES X CONTROLLER</h4>
+                                    <span>₹7999 </span>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="game">
+            <div class="container">
+                <div class="what-game text-center">
+                    <h2>What Gamers Offer !</h2>
+                </div>
+                <div class="row">
+                    <div class="col-md-3">
+                        <div class="layer-2 text-center">
+                            <img src="img/free-delivery 1.png" alt="" />
+                            <h6 class="">Free Shiping</h6>
+                            <p>
+                                Lorem ipsum dolor sit amet, consectetur
+                                adipiscing elit. Massa purus gravida.
+                            </p>
+                        </div>
+                    </div>
+                    <div class="col-md-3">
+                        <div class="layer-2 text-center">
+                            <img src="img/cashback 1.png" alt="" />
+                            <h6 class="">Exclusiv Deals</h6>
+                            <p>
+                                Lorem ipsum dolor sit amet, consectetur
+                                adipiscing elit. Massa purus gravida.
+                            </p>
+                        </div>
+                    </div>
+                    <div class="col-md-3">
+                        <div class="layer-2 text-center">
+                            <img src="img/premium-quality 1.png" alt="" />
+                            <h6 class="">Quality Product</h6>
+                            <p>
+                                Lorem ipsum dolor sit amet, consectetur
+                                adipiscing elit. Massa purus gravida.
+                            </p>
+                        </div>
+                    </div>
+                    <div class="col-md-3">
+                        <div class="layer-2 text-center">
+                            <img src="img/24-hours-support 1.png" alt="" />
+                            <h6 class="">24/7 Support</h6>
+                            <p>
+                                Lorem ipsum dolor sit amet, consectetur
+                                adipiscing elit. Massa purus gravida.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="products">
+            <div class="container text-center">
+                <div class="trending">
+                    <h2>Trending Products</h2>
```

Comment:
Space between sections are not like the design

Reply:
Done